imagersite/imager_profile/views.py

```diff
@@ -35,23 +35,15 @@ def get_context_data(self, **kwargs):
 
 
 class ProfileEditView(LoginRequiredMixin, UpdateView):
+    """View for updating user profile."""
     template_name = 'imager_profile/profile_edit.html'
     model = ImagerProfile
-    form_class = ProfileEditForm
     login_url = reverse_lazy('auth_login')
     success_url = reverse_lazy('profile')
     slug_url_kwarg = 'username'
     slug_field = 'user__username'
```

Comment:
Since you are overwriting the `get_object()` method, you do not need to specify the `slug_field` and `slug_url_kwarg`.